src/components/Lending/utils.tsx

```diff
@@ -0,0 +1,27 @@
+import { BigNumber } from 'bignumber.js'
```

Comment:
this file is a duplicate and is also found under rewards, moving it to a utils folder so both modules can import this will be better